schemas/generator.schemas.js

```diff
@@ -0,0 +1,36 @@
+import { resolve } from 'path';
+import * as TJS from 'typescript-json-schema';
+import * as fs from 'fs';
+
+// Arguments to be passed to schema generator
+const settings = {
+    noExtraProps: true,
+    aliasRefs: true
+};
+
+const schemasToGenerate = [
+    {
+        filePath: './question.ts',
+        typeName: 'Quizoot.Question',
+        schemaName: 'question'
+    },
+    {
+        filePath: './quiz.ts',
+        typeName: 'Quiz',
+        schemaName: 'quiz'
+    }
+];
+
+schemasToGenerate.forEach((obj) => {
+    const program = TJS.getProgramFromFiles([resolve(obj.filePath)]);
+
+    const schema = TJS.generateSchema(program, obj.typeName, settings);
+
+    // TODO: Write the schema in a .json file
```

Comment:
No idea on how to retrieve the schema data in a file...
This is not working with `fs.writeFile`

Reply:
Let me take a look. I prefer Solution 3 as well